src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl

```diff
@@ -14897,3 +14897,41 @@ voag:QUDT-QUANTITY-KINDS-VocabCatalogEntry_v1.2
   rdfs:label "QUDT Quantity Kinds Vocabulary Catalog Entry v1.2" .
 
 
+# new quantity kinds
+
+quantitykind:spectralIrradiance
+  a qudt:QuantityKind ;
+  rdfs:label "Measurement Unit of Spectral Irradiance";
+  qudt:plainTextDescription "Spectral Irradiance is a radiometry term for the power per unit area of electromagnetic radiation per unit wavelength incident at a surface." ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
+  qudt:hasDimensionVector qkdv:A0E0L-4I0M1H0T-3D0 .
+
+quantitykind:spectralEmittance
+  a qudt:QuantityKind ;
+  rdfs:label "Measurement Unit of Spectral Emittance";
+  qudt:plainTextDescription "Spectral Emittance is a radiometry term for the power per unit area of electromagnetic radiation unit wavelength emerging from a surface." ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
+  qudt:hasDimensionVector qkdv:A0E0L-4I0M1H0T-3D0 .
+  
+quantitykind:spectralRadiance
+  a qudt:QuantityKind ;
+  rdfs:label "Measurement Unit of Spectral Radiance";
+  qudt:plainTextDescription "Spectral Radiance is a radiometry term for the power per unit area of electromagnetic radiation per unit wavelength per unit solid angle emerging from a surface." ;
+  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D-1;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> .
+
+quantitykind:thermalInertia
+  a qudt:QuantityKind ;
+  rdfs:label "Measurement Unit of Thermal Inertia";
+  qudt:PlainTextDescription "Thermal inertia is a term used to describe the observed delays in a body's temperature response during heat transfers";
```

Comment:
Should be qudt:plainTextDescription, not qudt:PlainTextDescription.

Reply:
fixed in 15d1360fefa7f2bec2ba499ee6f4f217a005a85e